neoload/commands/test_settings.py

```diff
@@ -169,26 +169,17 @@ def fill_default_fields(json_data):
     ])
     return data
 
-def createorpatch(name, rename, description, scenario, controller_zone_id, lg_zone_ids, naming_pattern):
-    __id = None
 
+def create_or_patch(name, rename, description, scenario, controller_zone_id, lg_zone_ids, naming_pattern):
     is_id = tools.is_id(name)
 
-    try:
-        __id = tools.get_id(name, __resolver, is_id)
-        logging.info('Found test-setting: ' + __id)
-    except cli_exception.CliException as err:
-        if 'no id associated' not in err.message.lower():
-            raise err
+    __id = tools.get_id(name, __resolver, is_id, True)
+    logging.info('Found test-setting: ' + __id)
```

Comment:
This line throws an exception when the test does not exist:
TypeError: can only concatenate str (not "NoneType") to str
Because __id is None. This should be :
>>  logging.info('Found test-setting: ' + str(__id))